package.json

```diff
@@ -4,8 +4,12 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon app.js"
+    "start": "node server/app.js",
+    "server": "nodemon node server/app.js",
```

Comment:
you dont need the second `node` here :sweat_smile:

Reply:
I kinda do!! the first one is for heroku, the second one is for development mode